README.md

```diff
@@ -14,21 +14,26 @@ The app is currently in the second stage of development and several new features
 
 The project uses test-drive agile development processes, where requirements are turned into test cases, and code is improved upon. 
 
+#### Table of Contents
+* [👾 Technologies](##👾Technologies)
+* [📸  Screenshots](##📸 Screenshots)
+
+
 [<img src="https://github.com/pushpinderpalsingh/SpaceDash/blob/develop/Screenshots/Download_on_the_App_.png"  width=25%>](https://apps.apple.com/in/app/space-dash/id1527766640)
 
-## [](https://github.com/pushpinderpalsingh/SpaceDash#screenshots)Technologies 👾 
-> To help scale development this app is built using the MVC design pattern, changes made by contributors subsequently do not affect the entire model
+## 👾 Technologies
+> To help scale development, this app is built using the MVC design pattern. subsequent changes made by contributors do not affect the central architecture.
 * Fetches and renders JSON from the SpaceX API
 * Intuitive error handling
 * Animations via the Lottie framework
 * Unit testing using XCTests framework
 
 
-## [](https://github.com/pushpinderpalsingh/SpaceDash#screenshots)Screenshots 📸 
+## 📸 Screenshots
 
 [![alt text](https://github.com/pushpinderpalsingh/SpaceX/raw/develop/Screenshots/collage.png "Screen Shots")](https://github.com/pushpinderpalsingh/SpaceX/blob/develop/Screenshots/collage.png)
 
-## [](https://github.com/pushpinderpalsingh/SpaceDash#getting-started-for-development)Getting Started With Development ⚙️
+## ⚙️ [](https://github.com/pushpinderpalsingh/SpaceDash#getting-started-for-development)Quick Start
```

Comment:
I believe the previous title was perfect.

Reply:
I'm not convinced. "Getting started with Development" sounds like an introduction to the career of being a developer. It doesn't sound like native English in this context. This is getting started using our project Space Dash as a developer, therefore we could be more specific.

I see "Quick Start", often used as a reference to the expression "quickly start your development environment". The information is all about getting the project up and running.

Reply:
Then rename to just Getting Started.